example/App.js

```diff
@@ -43,6 +43,10 @@ export default class App extends Component<Props> {
         headers: {}
       },
       
+      //apkHeaders is optional and it is an object of headers to be passed to the server
+      //You should use it if you need to pass headers besides the url (apkUrl from test-version.json) to download the new APK
+      apkHeaders: {},
```

Comment:
Can we make the "shape" of the object here match the previous PR you made for 'apkVersionOptions'? That way the objects could maybe even be reused but either way the names and object you pass in would harmoize

```suggestion
      apkOptions: {
        method: 'GET',
        headers: {}
      },
```

Reply:
Good idea!

I changed apkHeaders to apkOptions, but apkOptions does not have the same parameters as apkVersionOptions. 

ApkVersionOptions is an options object for the fetch method while apkOptions receives the options (type DownloadFileOptions) from the DownloadFile function of react-native-js